packages/pigeon/lib/pigeon_lib.dart

```diff
@@ -196,6 +203,7 @@ options:
         help: 'REQUIRED: Path to generated dart source file (.dart).')
```

Comment:
In the helps, write what the default values are

Reply:
There are no default values, it's required.